<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModbusMaster</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonOpenPort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>5</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClosePort">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>5</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="typeOfParityChek">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>162</y>
      <width>144</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип проверки на четность</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxParity">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>162</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="exchangeRate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>106</y>
      <width>97</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Скорость обмена</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxStopBits">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxDataBits">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>134</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxBaudRate">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>106</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="listOfPorts">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>78</y>
      <width>86</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Список портов:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="numberOfBitsPerBite">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>134</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Число бит в байте</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPort">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>78</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="numberOfStopBits">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>113</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Число стоповых бит</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="startRegisterLineEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="numberOfRegistersLineEdit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelStartReg">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Стартовый регистр</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelNumberReg">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>150</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество регистров</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="sendRequestButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>230</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Request </string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>201</width>
      <height>231</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Номер регистра</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Значение</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="comboBoxFormat">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>230</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Формат отображения</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>826</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
